docs/python_docs/python/tutorials/packages/gluon/loss/loss.md

```diff
@@ -160,7 +160,7 @@ In classification, we often apply the
 softmax operator to the predicted outputs to obtain prediction probabilities,
 and then apply the cross entropy loss against the true labels:
 
-$$ \begin{align}\begin{aligned}p = \softmax({pred})\\L = -\sum_i \sum_j {label}_j \log p_{ij}\end{aligned}\end{align}
+$$ \begin{align}\begin{aligned}p = softmax({pred})\\L = -\sum_i \sum_j {label}_j \log p_{ij}\end{aligned}\end{align}
```

Comment:
Think latex wise using `\text{softmax}` is better that plain "softmax".

Reply:
Isn't softmax the name of the function, not a LaTex command? I made this change because a ticket pointed out that there was a LaTex rendering issue here: https://mxnet.apache.org/api/python/docs/tutorials/packages/gluon/loss/loss.html#Cross-Entropy-Loss-with-Softmax

Reply:
What you did works. Previously it was `\softmax` with the rendering issue, but enclosing it in `\text{softmax}` tells latex to properly handle text written in latex.